### PR TITLE
ZP-120 : Create account types for Agri, Professional, Hospital

### PR DESCRIPTION
**Pull request by nbhusare**
1. Created implementations for various account types
2. Also modified the CurrentAccountClass with setter and getter for customer

### CODE REVIEW

File: com.zafin.demo.bank/src/com/zafin/demo/bank/account/CurrentAccount.java [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+
 public class CurrentAccount {
+	
+	private Customer customer;
+
+	public Customer getCustomer() {
+		return customer;
+	}
 
+	public void setCustomer(Customer customer) {
```
Comment (pkgoddard) [inline com.zafin.demo.bank/src/com/zafin/demo/bank/account/CurrentAccount.java]: Let's make this class immutable by removing setters.

File: com.zafin.demo.bank/src/com/zafin/demo/bank/account/CurrentAccount.java [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+	
+	private final Customer customer;
+
+	public CurrentAccount(Customer customer) {
+		this.customer = customer;
+	}
+	
+	public Customer getCustomer() {
+		return customer;
+	}
```
Comment (pkgoddard) [inline com.zafin.demo.bank/src/com/zafin/demo/bank/account/CurrentAccount.java]: Nice job!

**Comment by pkgoddard** [comment]
LGTM

**Comment by nbhusare** [comment]
LGTM

**Pull request merged**